doc/generic/pgf/text-en/pgfmanual-en-tutorial.tex

```diff
@@ -471,6 +471,11 @@ \subsection{Adding a Touch of  Style}
 \end{tikzpicture}
 \end{codeexample}
 
+ In this example, the definition of the style "Karl's grid" is given as an 
```

Comment:
```suggestion
 In this example, the definition of the style ``Karl's grid'' is given as an 
```
Please use the correct quotes.  What you have typed right now will render absolutely horribly.  GitHub is even highlighting it in red.
Other than that, it looks good to me.

Reply:
Wouldn't a monospace (code) font be better? 
```suggestion
 In this example, the definition of the style |Karl's grid| is given as an 
```

Reply:
Probably yes.